image-resize/file.js

```diff
@@ -0,0 +1,25 @@
+const fs = require('fs');
+
+// Promisify file read
+exports.read = (filename) => {
+  return new Promise((resolve,reject) => {
+    fs.readFile(filename, (error, data) => {
+      if(error) {
+        reject(error);
+      }
+      resolve(data);
+    });
+  });
+};
+
+// Promisify file write
+exports.write = (filename, data) => {
+  return new Promise((resolve,reject) => {
+    fs.writeFile(filename, data, error => {
```

Comment:
Same with `writeFile`. 

Also make all of these functions `async` and get rid of these raw promises.